cmd/deploy/deploy.go

```diff
@@ -43,7 +43,7 @@ const (
 )
 
 var (
-	deployRemote = false
```

Comment:
Could we move this variable to deployCommand instead of having it global

Reply:
done!